recipes/installer.rb

```diff
@@ -33,18 +49,10 @@
   else
     raise "Unknown package type encountered for install: #{File.extname(remote_path)}"
   end
-  action :nothing
   if(node[:omnibus_updater][:restart_chef_service])
     notifies :restart, resources(:service => 'chef-client'), :immediately
   end
   notifies :create, resources(:ruby_block => 'omnibus chef killer'), :immediately
-end
-
-ruby_block 'Omnibus Chef install notifier' do
-  block{ true }
-  action :nothing
-  subscribes :create, resources(:remote_file => "omnibus_remote[#{File.basename(remote_path)}]"), :immediately
-  notifies :run, resources(:execute => "omnibus_install[#{File.basename(remote_path)}]"), :delayed
   only_if { node['chef_packages']['chef']['version'] != node['omnibus_updater']['version'] }
 end
```

Comment:
Does this change cause the `omnibus_install` execute resource to always re-install on every `chef-client` run if the `omnibus_updater/version` attribute is `nil` (the default that means "install the latest")?